testing/dockerfiles/baz/Dockerfile.tmpl

```diff
@@ -0,0 +1,5 @@
+FROM    ubuntu
+RUN     apt-get update && apt-get install -y wget
+RUN     wget -q -O /dummy.deb ${IP_ADDRESS}/dummy.deb || true
```

Comment:
Instead of a template, just use `ARG`

Reply:
Not sure why I didn't think of using an ARG when that's literally what I wanted.